apps/rotm/behaviours/back-link-check-report.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  getBackLink(req, res) {
+    let backLink = super.getBackLink(req, res);
```

Comment:
Where is `super.getBackLink`? Is `suer.getBackLink` sync or async? Can you point me to it please?

Reply:
looks like it's sync 
https://github.com/UKHomeOfficeForms/hof-form-controller/blob/master/lib/controller.js#L83

does that mean in this example the callback was not required
https://github.com/UKHomeOffice/firearms/blob/ced343a97023bac42aea7e45ff4b5f568154ad5c/apps/common/controllers/address-loop.js#L15